src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todoList }) => (
+  <section className="TodoList">
+    {todoList.map(todo => (
+      <TodoInfo todo={todo} key={todo.id} />
+    ))}
+  </section>
```

Comment:
please use semantic tags (`ul, li`) for this list

Reply:
i.e. you need to write "ul" instead of a tag "section" and "li" instead of "article"?